src/geometric_algebra/generic/inspirations.lean

```diff
@@ -0,0 +1,33 @@
+/-
+Copyright (c) 2020 lean-ga Team. All rights reserved.
+Released under MIT license as described in the file LICENSE.
+Authors: Utensil Song
+
+This file contains only imports which are just the lean files
+that the authors draw inspirations from.
```

Comment:
The goal here is a quick "jump to definition"?

Reply:
Yes, only that. Since this is a branch so I'm a little casual about file organization and leave it by the side of the files I'm working on.

Reply:
I like this idea, just would like a comment in this file pointing out it it can be used this way :)